code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -69,7 +68,7 @@
 	desc = "The three birds united their efforts to defeat the beast. \
 	This could stop countless incidents, but you’ll have to be prepared to step into the Black Forest…"
 	icon_state = "twilight"
-	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 90, BLACK_DAMAGE = 90, PALE_DAMAGE = 90) // 360
+	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 80, BLACK_DAMAGE = 80, PALE_DAMAGE = 50) // 290
```

Comment:
It is planned to give twilight armor an ability(like in LC), so there's no need to change it until it's done.

Reply:
I can revert this one, but Paradise lost is NOT okay.